edit/location.go

```diff
@@ -139,6 +139,20 @@ func (loc *location) Accept(i int, ed *Editor) {
 	ed.mode = &ed.insert
 }
 
+//This removes the $pwd from the current directory-historyDue to the way the paths are handled before this operation , we have to get all paths evaluated to the homedir , which gives
```

Comment:
You can just add pwd to the blacklist on line 164. There is no need to convert the directory values to absolute paths either; the directory entries in storage always contain results from `os.Getwd` (c.f. https://github.com/elves/elvish/blob/master/eval/chdir.go).

Reply:
I'll look into it tomorrow morning . Thanks for the warm welcome :smile: .

Reply:
Okay , I figured out why on earth my initial commit wasn't working as intended when I started trying it out at home . So : util.Getwd() -- the function that returns just the string of the pwd , returns it as it gets displayed : as "~/go/src/elves/elvish" but ; os.Getwd() returns it as "/home/user/go/src/elves/elvish" . I was trying to put util.Getwd() in the blacklist while the directory listing had os.Getwd() history in it .
In the current form , I would overwrite the current blacklist/pinned list .
@xiaq How would one blacklist a folder ?
 I'll resubmit a full solution tomorrow morning .